### PR TITLE
I1135 proxy

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

**Comment by richfitz** [comment]
Amazing I managed to miss this one :unamused:

**Comment by hillalex** [comment]
Actually it was silly of me to create this PR because now I can't review it, but it works nicely for me when I run Montagu locally pointing at this branch of the shiny submodule (and set the cookie manually by pulling from a portal request) Lets talk through this tomorrow though to make sure I've understood it!

**Comment by richfitz** [comment]
Oh of course!  How odd!

File: nginx/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+  shiny:
+    image: docker.montagu.dide.ic.ac.uk:5000/montagu-shiny:i1135_proxy
+    # ports:
+    #   - "3838:3838"
+  shiny_proxy:
+    image: docker.montagu.dide.ic.ac.uk:5000/montagu-shiny-proxy:i1135_proxy
+    # ports:
+    #   - "1080:80"
```
Comment (hillalex) [inline nginx/docker-compose.yml]: remove these commented lines

File: nginx/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  server {
+    listen 80;
+
+    location /shiny/ {
+      rewrite ^/shiny/(.*)$ /$1 break;
+      # So this is the bit that needs work:
+      #
+      # changed localhost:3838 to shiny:3838
```
Comment (hillalex) [inline nginx/nginx.conf]: remove comments

**Pull request merged**